tests/functional/aws-node-sdk/test/object/initiateMPU.js

```diff
@@ -52,6 +52,14 @@ describe('Initiate MPU', () => {
              })
         );
 
+        it('should return KeyTooLong error when key is longer than 915 bytes', done =>
+            s3.createMultipartUpload({ Bucket: bucket, Key: 'a'.repeat(916) }, err => {
+                assert.strictEqual(err.code, 'KeyTooLong');
```

Comment:
check not null before, to void accessing field of null :
```suggestion
				assert.notEqual(err, null);
                assert.strictEqual(err.code, 'KeyTooLong');
```

Reply:
or may use `checkError`